integrations/access/msteams/configure.go

```diff
@@ -145,7 +131,18 @@ func copyAssets(targetDir string) error {
 	return nil
 }
 
-func configureAppZip(targetDir, fileName string) error {
+func ConfigureAppZip(targetDir, fileName string, p Payload) error {
+	manifestWriter, err := os.Create(path.Join(targetDir, "manifest.json"))
```

Comment:
```suggestion
	manifestWriter, err := os.Create(filepath.Join(targetDir, "manifest.json"))
```

Use the `filepath` package instead of `path` when you're working with filepaths. This ensures your code will work on filesystems that use different path separators.